@navikt/core/css/modal.css

```diff
@@ -128,6 +128,18 @@
   overflow: auto;
   overscroll-behavior: contain;
   position: relative; /* Needed to make sr-only elements position correctly - see Storybook */
+  background:
+    linear-gradient(white 30%, transparent) top,
+    linear-gradient(transparent, white 60%) bottom,
```

Comment:
Would it be possible to use `inherit` here instead of `white` in case the user changed the background?

Reply:
`inherit` doesn't work.

Reply:
Not optimal, but works with tokens atleast:
```
.navds-modal {
  --__ac-modal-bg: var(--ac-modal-bg, var(--a-surface-default));

  background-color: var(--__ac-modal-bg);
  border: none;
  border-radius: var(--a-border-radius-large);
  box-shadow: var(--a-shadow-xlarge);
  padding: 0;
  position: fixed;
  max-height: 100%;
  max-width: 100%;
  color: var(--a-text-default);
}

.navds-modal__body {
  padding: var(--a-spacing-6);
  overflow: auto;
  overscroll-behavior: contain;
  position: relative; /* Needed to make sr-only elements position correctly - see Storybook */
  background:
    linear-gradient(var(--__ac-modal-bg) 30%, transparent) top,
    linear-gradient(transparent, var(--__ac-modal-bg) 60%) bottom,
    radial-gradient(farthest-side at 50% 0, rgba(0 0 0 / 0.3), transparent) top,
    radial-gradient(farthest-side at 50% 100%, rgba(0 0 0 / 0.3), transparent) bottom;
  background-size:
    100% 40px,
    100% 50px,
    100% 16px,
    100% 16px;
  background-attachment: local, local, scroll, scroll;
  background-repeat: no-repeat;
}
```

Reply:
Nice! I totally forgot that we had a token for bg color 😂